packages/multi-select-combo-box/theme/lumo/vaadin-multi-select-combo-box-chip-styles.js

```diff
@@ -13,27 +13,65 @@ import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themab
 
 const chip = css`
   :host {
-    display: inline-flex;
-    align-items: center;
-    align-self: center;
     font-family: var(--lumo-font-family);
     font-size: var(--lumo-font-size-xxs);
     line-height: 1;
-    padding: 0.3125em 0 0.3125em calc(0.5em + var(--lumo-border-radius-s) / 4);
+    padding: 0.3125em calc(0.5em + var(--lumo-border-radius-s) / 4);
     border-radius: var(--lumo-border-radius-s);
     border-radius: var(--lumo-border-radius);
     background-color: var(--lumo-contrast-20pct);
     cursor: var(--lumo-clickable-cursor);
-    white-space: nowrap;
-    box-sizing: border-box;
-    min-width: 0;
+  }
+
+  :host(:not([part~='overflow'])) {
+    padding-inline-end: 0;
+  }
+
+  :host([part~='overflow']) {
+    position: relative;
+    min-width: var(--lumo-size-xxs);
+    margin-inline-start: var(--lumo-space-s);
+  }
+
+  :host([part~='overflow'])::before,
+  :host([part~='overflow'])::after {
+    position: absolute;
+    content: '';
+    width: 3px;
+    height: 21px;
+    border-left: 2px solid;
+    border-radius: 4px 0 0 4px;
+    border-color: var(--lumo-contrast-30pct);
+  }
+
+  :host([part~='overflow'])::before {
+    left: -4px;
+  }
+
+  :host([part~='overflow'])::after {
+    left: -8px;
+  }
+
+  :host([part~='overflow-two']) {
+    margin-inline-start: calc(var(--lumo-space-s) / 2);
+  }
+
+  :host([part~='overflow-two'])::after {
+    display: none;
+  }
+
+  :host([part~='overflow-one']) {
+    margin-inline-start: 0;
+  }
+
+  :host([part~='overflow-one'])::before,
+  :host([part~='overflow-one'])::after {
+    display: none;
   }
 
   [part='label'] {
     color: var(--lumo-body-text-color);
     font-weight: 500;
-    overflow: hidden;
-    text-overflow: ellipsis;
     line-height: 1.25;
```

Comment:
I’d let these styles inherit from the input itself. I don't see the need to re-assign them. Also, drop the line-height definition, and instead increase the vertical padding on the host to `0.5em`.

Reply:
Removed line-height.

Reply:
As discussed in Slack, we need to keep the line-height, to avoid font descenders from getting clippped.